### PR TITLE
Icebox layer deletion

### PR DESCRIPTION
**Pull request by daniviga**
This PR will trigger layer and maps removal when an Icebox calculation is deleted. It will also DROP the database view associated with the icebox output.

We are missing a migration to add layers previously produced by Icebox in the "icebox" layers group. This will be part of a separate PR.

Thanks @ptormene for the peering.

### CODE REVIEW

**Comment by daniviga** [comment]
Migration: https://github.com/gem/oq-platform/pull/377

File: openquakeplatform/openquakeplatform/icebox/models.py [hunk 1]
```diff
@@ -154,8 +155,16 @@ def create_geonode_map(self, layers):
...
 
     @staticmethod
-    def remove_map(sender, instance, using, **_kwargs):
+    def remove_calc(sender, instance, using, **_kwargs):
         if instance.map_id:
+            for layer in instance.map.layer_set.all():
+                if layer.group == 'icebox':
+                    layer_obj = Layer.objects.get(typename=layer.name)
+                    cursor = connection.cursor()
+                    view_name = layer.name.split(":")[1]
```
Comment (ptormene) [inline openquakeplatform/openquakeplatform/icebox/models.py]: We might add a comment with an example like geonode:layername

**Comment by ptormene** [comment]
Apart from the minor note, LGTM

**Comment by ptormene** [comment]
LGTM

**Pull request merged**